loader/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
-sleep 10s;
+#sleep 10s;
```

Comment:
Have you commented that only for testing?

Reply:
I believe we don't need these anymore